pkg/resources/task_acceptance_test.go

```diff
@@ -157,6 +157,8 @@ var (
 )
 
 func Test_AccTask(t *testing.T) {
+	t.Skip("broken by a change to snowflake")
```

Comment:
Would you be able to link to a test with broken results or add some context(either here or the pr) so whomever works on fixing this can get a head start?